packages/test/test-end-to-end-tests/src/test/serializeAfterFailedAttach.spec.ts

```diff
@@ -38,10 +33,16 @@ describeCompat(
 		};
 		const sharedStringId = "ss1Key";
 		const sharedMapId = "sm1Key";
-		async function createDetachedContainerAndGetEntryPoint(
-			loader: IHostLoader,
-			request: IRequest,
-		) {
+		async function createDetachedContainerAndGetEntryPoint(provider: ITestObjectProvider) {
```

Comment:
nit: now that the failure injection (and not just the expectations) happens in this function, `createAndFailToAttachContainer` seems like a more fitting name.